scripts/ecs/appsignals/sample-app/task-definitions/spring-petclinic-admin-server.json

```diff
@@ -0,0 +1,53 @@
+{
+  "family": "admin-server",
+  "networkMode": "awsvpc",
+  "requiresCompatibilities": [
+    "FARGATE"
+  ],
+  "cpu": "256",
+  "memory": "512",
+  "containerDefinitions": [
+    {
+      "name": "admin-server",
+      "image": "admin-server-image",
+      "portMappings": [
+        {
+          "containerPort": 9090,
+          "protocol": "tcp"
+        }
+      ],
+      "environment": [
+        {
+          "name": "CONFIG_SERVER_URL",
+          "value": "http://config-server-cluster-name.ecs-petclinic:8888"
+        },
+        {
+          "name": "SPRING_PROFILES_ACTIVE",
+          "value": "prod"
+        },
+        {
+          "name": "DISCOVERY_SERVER_URL",
+          "value": "http://discovery-server-cluster-name.ecs-petclinic:8761/eureka"
+        },
+        {
+          "name": "ADMIN_IP",
+          "value": "admin-server-cluster-name.ecs-petclinic"
+        }
+      ],
+      "essential": true,
+      "logConfiguration": {
+        "logDriver": "awslogs",
+        "options": {
+          "awslogs-group": "/ecs/pet-clinic-admin-server",
+          "awslogs-create-group": "true",
+          "awslogs-region": "us-west-2",
```

Comment:
Remove hard-coded regions. Same applies to all the other log group configuration.

Reply:
replaced with `"region-name"`